README.md

```diff
@@ -48,24 +53,29 @@ minio server /data --console-address ":9001"
 
```

Comment:
The first item in the list (download deps) doesn't match the instructions (navigate to folder).

Also, it isn't quite clear where to navigate, because if I clone the repo, there will be `tus_storagehandler/` (the name of the repo) and `tus_storagehandler/tus_storagehandler/` (the name of the package). Please include an additional step (step 1) where you explain how to clone the repository, then add the navigation step (step 2). For both, include the actual line of code that people need to execute (`git clone git@github.com:elixir-cloud-aai/tus-storagehandler.git` and `cd tus_storagehandler`), respectively.

Then continue with "Install Poetry" (now step 3).